types/simpleheat/index.d.ts

```diff
@@ -0,0 +1,76 @@
+// Type definitions for simpleheat 0.4
+// Project: https://github.com/mourner/simpleheat
+// Definitions by: Tristan de Guillebon <https://github.com/tristangbn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = simpleheat;
+export as namespace simpleheat;
+
+declare class simpleheat {
+    /**
+     * Create a new simpleheat instance.
+     */
+    constructor(canvas: HTMLCanvasElement);
```

Comment:
this would work ok, but the module allows old pattern with a creator function,
in order to support this, you can consider something along lines:
```ts
declare namespace simpleheat {
    interface Instance {
        data(points: Array<[number, number] | [number, number, number]>): this;
        ...
    }
    interface Static {
        new(canvas: HTMLCanvasElement): Instance;
        (canvas: HTMLCanvasElement): Instance;
    }
}

declare const simpleheat: simpleheat.Static;

export = simpleheat;
export as namespace simpleheat;
```
(you can adapt Instance/Static naming to your needs)

Reply:
Hi! Thank you very much! Didn't know how to deal with this case...
I updated the commit with your feedbacks ;)